draft-ietf-webpush-protocol.xml

```diff
@@ -327,6 +338,18 @@ Host: push.example.net
         that these URIs are not disclosed to unauthorized recipients (see <xref
         target="authorization"/>).
       </t>
+      <t>
+        The push service MAY provide a URI for a subscription set resource in a
+        link relation of type "urn:ietf:params:push:set". If provided, the push
+        service supports subscription sets. If available, the user agent SHOULD

```

Comment:
As an additional gotcha if its SHOULD, a client may ask for a subscription set and an individual subscription in the set at the same time, which could result in some odd situations where its likely the client will get dupes of messages since the server could dump what it has under both overlapping concerns before any ack's are received.